back/app/finders/events_finder.rb

```diff
@@ -14,11 +14,11 @@ def project_publication_statuses_condition(project_publication_statuses)
     scope(:with_project_publication_statuses, project_publication_statuses) if project_publication_statuses.present?
   end
 
-  def start_at_lt_condition(start_at)
-    where('start_at < ?', start_at)
+  def end_at_lt_condition(end_at)
+    where('end_at < ?', end_at)
   end
 
-  def start_at_gteq_condition(start_at)
-    where('start_at >= ?', start_at)
+  def end_at_gteq_condition(end_at)
```

Comment:
Are these method names 'good practice'?

Reply:
I don't think so. They felt weird to both me and Alex.

I think better would be something more descriptive like `get_past_events` and `get_current_and_futue_events`